app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -203,31 +208,66 @@ class HomeFragment : Fragment() {
         }
     }
 
+    private fun setupPrivateBrowsingDescription() {
+        // Format the description text to include a hyperlink
+        val descriptionText = String
+            .format(private_session_description.text.toString(), System.getProperty("line.separator"))
+        val linkStartIndex = descriptionText.indexOf("\n\n") + 2
+        val linkAction = object : ClickableSpan() {
+            override fun onClick(widget: View?) {
+                // TODO Go to SUMO page
+            }
+        }
+        val textWithLink = SpannableString(descriptionText).apply {
+            setSpan(linkAction, linkStartIndex, descriptionText.length, 0)
+
+            val colorSpan = ForegroundColorSpan(private_session_description.currentTextColor)
+            setSpan(colorSpan, linkStartIndex, descriptionText.length, 0)
+        }
+        private_session_description.text = textWithLink
+    }
+
+    private fun displayPrivateSessionDescriptionIfNeeded() {
```

Comment:
small naming nit: perhaps something like `updatePrivateSessionDescriptionVisibility` since this function can actually *hide* the description--not just make it visible when necessary.

Reply:
good catch. After some discussion I think we'll quickly pull this out of the HomeFragment as well :+1: